.golangci.yml

```diff
@@ -1,5 +1,27 @@
+linters:
+  disable-all: true
```

Comment:
I'd suggest to flip these. Enable all linters and explicitly disable the others, so we know which ones we are disabling and folks can pick up the task of removing a disabled linter from this list if they wanted to.

Reply:
This is explicitly called out here because enable-all is deprecated  

https://github.com/golangci/golangci-lint/blob/b9eef79121fff235d0d794c176ffa2b3d9bd422f/.golangci.yml#L59-L60

Reply:
Oh well, it seems a bit counter intuitive to me, but nothing major